src/state/dune

```diff
@@ -13,8 +13,10 @@
 (rule
   (targets opamScript.ml)
   (deps    ../../shell/crunch.ml (glob_files shellscripts/*.*sh))
-  (action  (with-stdout-to %{targets} (run ocaml %{deps}))))
-
+  (action  (with-stdout-to %{targets}
+  (progn
+    (run ocaml %{deps})
+    (run sed -i -e "s/$SCRIPTOPAMVERSION\\b/%{read-lines:../version}/g" %{targets})))))
```

Comment:
`sed -i` isn't portable (macOS) and while unlikely "2.1.0" is itself a regex! This wants doing with an OCaml script using `Re` (it's tedious, but not too bad).

Reply:
The `sed -i %{targets}` within `with-stdout-to %{targets}` seems highly suspicious to me anyway :roll_eyes:

A cleaner approach would be to rename `bwrap.sh` to `bwrap.sh.in`, and have a separate rule for the sed from the latter to the former (I don't remember if `glob_files` will handle it correctly though).